sources/packages/api/src/utilities/date-utils.ts

```diff
@@ -199,3 +201,25 @@ export function getDateDifferenceInMonth(
 export const addDays = (date: Date | string, daysToAdd: number): Date => {
   return dayjs(date).add(daysToAdd, "day").toDate();
 };
+
+/**
+ * Return a PST timestamp with date and time in continuous format
+ * mainly used to append in filename.
+ * @param date
+ * @returns timestamp.
+ */
+export function getFileNameTimestamp(): string {
```

Comment:
since its formating the current timestamp. maybe you can give a generic name, something like `getCurrentTimeStamp` or something else